client/state/data-layer/wpcom/sites/automated-transfer/status/index.js

```diff
@@ -54,8 +54,11 @@ export const receiveStatus = ( { siteId }, { status, uploaded_plugin_slug, trans
 	}
 };
 
-export const requestingStatusFailure = ( { siteId } ) => {
-	return automatedTransferStatusFetchingFailure( siteId );
+export const requestingStatusFailure = ( response ) => {
+	return automatedTransferStatusFetchingFailure( {
+		siteId: response.siteId,
+		error: response.meta?.dataLayer?.error?.message || 'Unknown Error when fetching state',
```

Comment:
Are we going to use this default error for anything? If the answer is no, I'd suggest just refraining from adding it.

Reply:
But in case the backend error message is empty, isn't it better to have a verbose indication?

Reply:
Sure, but only if we use it for anything. It didn't seem to me like we used it for anything.

Reply:
My thinking was to fail loudly instead of failing silently, so that when investigating down the line we have a verbose error message instead of an empty string. 🤔

Reply:
That would make sense, but I wasn't sure where we really "fail loudly".  In my previous review, it looked to me like we're not using this error for anything, be it for the developer or for the user. Could you elaborate on this part?

Reply:
The message lives in the redux store for now, so that even on a redirect we can figure out what happened.

Reply:
This feels like polluting the redux tree to me TBH. It's not common practice to keep debug data into it just because we can. I'm not particularly fond of adopting this a common practice 😉

Reply:
Ok I will remove this :)